e2e-tests/tests/headlamp.spec.ts

```diff
@@ -16,3 +16,13 @@ test("headlamp is there and so is minikube", async ({ page }) => {
 
   await expect(page).toHaveURL(/.*main/);
 });
+
+test('GET /plugins/list returns plugins list', async ({ page }) => {
+  const response = await page.goto('/plugins/list');
```

Comment:
Should this be `/plugins` ? When I run the backend from this branch I get 404 on `/plugins/list`

Reply:
Weird, on web browser it gives the desired result

<img width="1792" alt="Screenshot 2023-05-09 at 7 31 59 AM" src="https://user-images.githubusercontent.com/24803604/236975294-9cff817c-5b30-4ace-a63b-e814a8e653cb.png">